main/docs/get-started/applications/wildcards-for-subdomains.mdx

```diff
@@ -63,8 +63,32 @@ The following restrictions apply when using the `{organization_name}` placeholde
 * A placeholder must not be prefixed nor suffixed with additional valid hostname characters. `https://prefix-{organization_name}-suffix.exampleco.com` will not work.
 * A placeholder **must not** be used in conjunction with a wildcard in the URL. `https://{organization_name}.*.exampleco.com` will not work.
 
+## Custom domain URL placeholders
```

Comment:
There are few things missing:
- mention these placeholders can be configured for the following Application URLs: Allowed Callback URLs, Allowed Logout URLs, Allows Web Origins, Allowed Origins (CORS)
- mention this does not work for Third Party Apps
- probably it is a good idea to link to https://auth0.com/docs/customize/custom-domains/multiple-custom-domains??